drivers/vmwarevsphere/flags.go

```diff
@@ -217,10 +224,14 @@ func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 	d.VAppProperties = flags.StringSlice("vmwarevsphere-vapp-property")
 	d.SetSwarmConfigFromFlags(flags)
 	d.ISO = d.ResolveStorePath(isoFilename)
+
 	d.CreationType = flags.String("vmwarevsphere-creation-type")
-	d.ContentLibrary = flags.String("vmwarevsphere-content-library")
+	if _, ok := creationTypes[d.CreationType]; !ok {
+		return fmt.Errorf("Creation type %s not supported", d.CreationType)
```

Comment:
This should be updated to say a creation type is required, as `d.CreationType` doesn't exist per the map check above.